<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>791</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancer Patient Management</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="chronic_lung">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Chronic Lung</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coughing_blood">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Coughing Blood</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level  :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="breath_shortness">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Breath Shortness</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chest_pain">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Chest Pain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="obesity">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Obesity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="level">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>high</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
